### PR TITLE
Bug/skale 3218 broadcast not found

### PR DESCRIPTION
**Pull request by DmytroNazarenko**

### CODE REVIEW

File: tools/bls/dkg_utils.py [hunk 1]
```diff
@@ -103,7 +103,10 @@ def broadcast_and_check_data(dkg_client, poly_name):
         logger.info(f'sChain {schain_name}: trying to receive broadcasted data,'
                     f'{RECEIVE_TIMEOUT - time_gone} seconds left')
 
-        events = dkg_filter.get_events()
+        if is_everyone_broadcasted(dkg_client):
```
Comment (olehnikolaiev) [inline tools/bls/dkg_utils.py:106]: what happen if someone doesn't send broadcast? `get_events` will never be called

Comment (DmytroNazarenko) [reply tools/bls/dkg_utils.py:106]: We use `get_events` with `from_channel_started_block` flag if all nodes have sent broadcast, but we haven't read any of them correctly. In other cases we'll use `get_events` without `from_channel_started_block`

**Review by olehnikolaiev — DISMISSED** [review]
great!

**Review by dmytrotkk — DISMISSED** [review]

**Review by olehnikolaiev — APPROVED** [review]
good!

**Review by dmytrotkk — APPROVED** [review]

**Pull request merged**